pymeasure/instruments/nf/li5660.py

```diff
@@ -0,0 +1,1665 @@
+        validator=strict_discrete_set,
+        values={"Off": "OFF",
+                "Expand": "EXP",
+                "Normalize": "NORM",
+                "Ratio": "RAT"},
+        map_values=True
+    )
+
+    ###############################
+    # Data subsystem, 24 Nov 2020 #
+    ###############################
+    buffer1_count = Instrument.measurement(
+        ":DATA:COUN? BUF1",
+        """ A property that allow user to queries the number of data sampling points recorded in the measurement data buffer1. """
+    )
+
+    buffer2_count = Instrument.measurement(
+        ":DATA:COUN? BUF2",
+        """ A property that allow user to queries the number of data sampling points recorded in the measurement data buffer2. """
+    )
+
+    buffer3_count = Instrument.measurement(
+        ":DATA:COUN? BUF3",
+        """ A property that allow user to queries the number of data sampling points recorded in the measurement data buffer3. """
+    )
+
+    data_timer = Instrument.control(
+        ":DATA:TIM?", ":DATA:TIM:STAT ON; :DATA:TIM %g",
+        """ A property that allow user to set the internal timer time interval. The time resolution is 640 ns """,
+        validator=truncated_range,
+        values=[1.92E-6, 20]
+    )
+
+    data_timer_state = Instrument.control(
+        ":DATA:TIM:STAT?", ":DATA:TIM:STAT %d",
+        """ A property that allow user to enable or disable the internal timer. """,
+        validator=strict_discrete_set,
+        values={"On": 1,
+                "Off": 0},
+        map_values=True
+    )
+
+    ##################################
+    # Display subsystem, 25 Nov 2020 #
+    ##################################
+    screen = Instrument.control(
+        ":DISP?", ":DISP %s", 
+        """There are 3 types of measurement screen display: Normal, Large and Fine Measurement screens""",
+        validator = strict_discrete_set,
+        values={"Normail": "NORM", "Large": "LARG", "Fine": "FINE"},
+        map_values=True
+    )
+
+    display_window = Instrument.control(
+        ":DISP:WIND?", ":DISP:WIND %d",
+        """ A property that allow user to on/off the display screen. """,
+        validator=strict_discrete_set,
+        values={"On": 1, "Off": 0},
+        map_values=True
+    )
+
+    ################################
+    # Fetch subsystem, 25 Nov 2020 #
+    ################################
+    fetch = Instrument.measurement(
+        ":FETC?",
+        """ Queries the most recent measurement data. """
+    )
+
+    #################################
+    # Format subsystem, 25 Nov 2020 #
+    #################################
+    form = Instrument.control(
+        ":FORM?", ":FORM %s",
+        """ A property that allow user to set the data transfer format. """,
+        validator=strict_discrete_set,
+        values={"ASCII": "ASC",
+                "Real": "REAL",
+                "Integer": "INT"},
+        map_values=True
+    )
```

Comment:
Do all data transfer formats properly work when getting the data using pymeasure (I presume with the `fetch` attribute)??